sensor_msgs/Image s_input
---
sensor_msgs/Image s_ouput